loris-logo.svg

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
there's already `htdocs/images/LORIS_logo.svg`, I don't think we should duplicate the image in the repo.